libs/community/tests/unit_tests/vectorstores/test_indexing_docs.py

```diff
@@ -55,6 +55,7 @@ def check_compatibility(vector_store: VectorStore) -> bool:
         "BigQueryVectorSearch",
         "Cassandra",
         "Chroma",
+        "CouchbaseVectorStore",
```

Comment:
we shoudl update the docs/docs/modules/data_connection/index.ipynb page to list this vector store

Reply:
Thanks, did not see that. I have added it now.